src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -1037,31 +1066,39 @@ private void WriteCrcAndSizesInLocalHeader(bool zip64HeaderUsed)
             // and setting the version to Zip64 to indicate that descriptor contains 64-bit values
             if (pretendStreaming)
             {
+                int relativeVersionToExtractLocation = ZipLocalFileHeader.FieldLocations.VersionNeededToExtract - ZipLocalFileHeader.FieldLocations.VersionNeededToExtract;
+                int relativeGeneralPurposeBitFlagsLocation = ZipLocalFileHeader.FieldLocations.GeneralPurposeBitFlags - ZipLocalFileHeader.FieldLocations.VersionNeededToExtract;
```

Comment:
I think these could be `const`, like the variables declared at the top of the method.

Reply:
Nevermind. If not possible, it's ok te leave them as is.